packages/teams-js/src/internal/communication.ts

```diff
@@ -389,7 +392,11 @@ function sendRequestToTargetWindowHelper(
 
   if (GlobalVars.isFramelessWindow) {
     if (Communication.currentWindow && Communication.currentWindow.nativeInterface) {
-      logger(`Sending message %i to ${targetWindowName} via framelessPostMessage interface`, request.uuidAsString);
+      logger(
+        'Sending message %s to %s via framelessPostMessage interface',
+        getMessageIdsAsLogString(request),
+        targetWindowName,
```

Comment:
Curious if you had a reason for passing input as arguments vs using an interpolated string?

Reply:
1. Consistency
2. I don't know if debug works this way but at least _some_ logger packages don't evaluate the arguments if they know that the log statement is being filtered, for performance. E.g., in this example, if `logger` knew that logging was disabled, it wouldn't call the `getMessageIdsAsLogString` function to get that value when it knew it didn't need it. Come to think of it now though, that would really be up to typescript where it evaluates all function arguments before actually calling the `logger` function, so this reason is completely irrelevant in TypeScript/JavaScript. So it's really just #1 then.